softlayer/disk/softlayer_creator.go

```diff
@@ -41,10 +41,21 @@ func (c SoftLayerCreator) Create(size int, virtualGuestId int) (Disk, error) {
 		return SoftLayerDisk{}, bosherr.WrapError(err, "Create SoftLayer Network Storage Service error.")
 	}
 
-	disk, err := storageService.CreateIscsiVolume(size, strconv.Itoa(vm.Datacenter.Id))
+	disk, err := storageService.CreateIscsiVolume(c.getSoftLayerDiskSize(size), strconv.Itoa(vm.Datacenter.Id))
 	if err != nil {
 		return SoftLayerDisk{}, bosherr.WrapError(err, "Create SoftLayer iSCSI disk error.")
 	}
 
 	return NewSoftLayerDisk(disk.Id, c.softLayerClient, c.logger), nil
 }
+
+func (c SoftLayerCreator) getSoftLayerDiskSize(size int) int {
+	sizeArray := []int{20, 40, 80, 100, 250, 500, 1000, 2000, 4000, 8000, 12000}
```

Comment:
This is where I would put a comment to explain why these are the only values for SL disk size. Maybe include a link to API docs. 

Keep comment small but this could be useful.